e2e/__tests__/detectOpenHandles.ts

```diff
@@ -71,6 +71,16 @@ it('does not report promises', () => {
   expect(textAfterTest).toBe('');
 });
 
+it('does not report ELD histograms', () => {
```

Comment:
look on line 100 in this file (`onNodeVersions`) - you wanna skip this test using that mechanism (with the correct minimum version), not via `try-catch` in the test itself